### ISSUE TITLE
CloudFormation support for AWS::Events::EventBus

### ISSUE THREAD

**Issue by harrydaniels-IW** [spulec/moto#3048]
## Feature Request

Hi, I'm trying to mock the creation of an AWS EventBus using Serverless framework and local stack, however it seems there is no Moto CloudFormation support for AWS::Events::EventBus resources.

### Serverless.yml:
```
resources:
  Resources:
    EventBus:
        Type: AWS::Events::EventBus
        Properties: 
          Name: New-Event-Bus
```

LocalStack uses the package: moto-ext>=1.3.15.12.

 ## Expected Results:

On deployment a new EventBus called "New-Event-Bus" is created.

**Comment by bblommers** [comment]
Thanks for raising this @harrydaniels-IW - marking it as an enhancement.

**Comment by bblommers** [comment]
This should be available in moto >= 1.3.15.dev887
